guide/14-deep-learning/unsupervised-ml-model-guide.ipynb

```diff
@@ -0,0 +1,618 @@
+{
```

Comment:
<p>it should be as a cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>